### PR TITLE
Fixed spelling mistakes in libcudf documentation

### PR DESCRIPTION
**Pull request by karthikeyann**
ran spell check and made some corrections.

### CODE REVIEW

**Review by ttnghia — APPROVED** [review]

**Comment by karthikeyann** [comment]
rerun tests

File: cpp/benchmarks/join/generate_input_tables.cuh [hunk 1]
```diff
@@ -169,7 +169,7 @@ void generate_input_tables(key_type* const build_tbl,
   // expense of not being that accurate with applying the selectivity an especially more memory
   // efficient implementations would be to partition the random numbers into two intervals and then
   // let one table choose random numbers from only one interval and the other only select with
-  // selectivity propability from the same interval and from the other in the other cases.
+  // selectivity probability from the same interval and from the other in the other cases.
```
Comment (davidwendt) [inline cpp/benchmarks/join/generate_input_tables.cuh:172]:
```suggestion
  // selective probability from the same interval and from the other in the other cases.
```
This whole sentence could use some work.

**Review by davidwendt — APPROVED** [review]
Technically behaviour is an acceptable spelling
https://grammarist.com/spelling/behavior-behaviour/
Regardless, it is good to have them all the same.
Thanks for doing this.

**Comment by karthikeyann** [comment]
@gpucibot merge

**Pull request merged**